src/main/java/classes/Storage.java

```diff
@@ -0,0 +1,117 @@
+package main.java.classes;
+
+import main.java.command.Deadline;
+import main.java.command.Event;
+import main.java.command.ToDo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Locale;
+import java.util.Scanner;
+
+/**
+ * Storage class to deal with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    private File file;
+
+    /**
+     * Constructor method.
+     * @param filePath path to the saved file
+     * @throws IOException if user IO is incorrect
+     */
+    public Storage(String filePath) throws IOException {
+        this.file = new File(filePath);
+        if (file.createNewFile()) {
+            System.out.println("file created!");
+        } else {
+            System.out.println("file loaded!");
+        }
+    }
+
+    /**
+     * Method to save Task in saved file, converts Task objects to Strings in file
+     * @param AL ArrayList of Task objects from user input
+     * @throws IOException if user IO is incorrect
+     */
+    public void saveTasks(ArrayList<Task> AL) throws IOException {
```

Comment:
Would AL be a bit confusing here? Perhaps `tasks` or `taskList` would be better?